docs/pages/docs/guides/beginners-guide.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Beginners guide
+description: A guide for beginners to onboard onto using Turborepo.
+---
+
+# Beginners guide
```

Comment:
- Rename to `Workspaces` 
- Add preamble/introduction to workspaces, with links and descriptions of how they work and how turbo is built on them

Reply:
Renamed to workspaces, added a section with links to external documentation, and changed the introduction slightly.
I believe this should be satisfactory. If not, let me know, and I'll apply changes as soon as I am able.